src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -73,7 +73,7 @@
     public void getSalaryInfoByMonth() throws Exception {
         try {
             Class<?> exceptionClass = Class
-                    .forName("core.basesyntax.exception.IllegalDateParametersException");
+                    .forName("core.basesyntax.core.basesyntax.exception.IllegalDateParametersException");
```

Comment:
don't change test

Reply:
Originalversion is "core.basesyntax.core.basesyntax.exception" :) it should be like this.

Reply:
Or I didn't pay attention and changed it.